.azure/bicep/config/web-app-settings.bicep

```diff
@@ -0,0 +1,45 @@
+param keyVaultName string
+param appInsightsName string
+param appName string
+param appSettings object = {}
+
+var secretRef = '@Microsoft.KeyVault(VaultName=${keyVaultName};SecretName='
+
+var sharedSettings = {
+  OutboundEmail__FromName: 'hutch'
+  OutboundEmail__Provider: 'sendgrid'
+  OutboundEmail__SendGridApiKey: '${secretRef}sendgrid-api-key)'
+  OutboundEmail__FromAddress: 'noreply@hutch.com'
+  // Environment for deployment (Development or Production)
+  ASPNETCORE_ENVIRONMENT: '${secretRef}ASPNETCORE_ENVIRONMENT)'
```

Comment:
Environment is definitely not a secret, and it's already passed in from the per environment files (e.g. `dev.bicep`)